src/webview/create-deployment/createDeploymentLoader.ts

```diff
@@ -243,7 +245,7 @@ export default class CreateDeploymentLoader {
                 action: 'getRecommendedBuilderImageStart'
             });
             analyzeRes = await Alizer.Instance.alizerAnalyze(uri);
-            normalizedBuilderImages = await BuilderImageWrapper.Instance.getBuilder();
+            normalizedBuilderImages = await (BuilderImageWrapper.getInstance() as BuilderImageWrapper).getBuilder();
```

Comment:
I think this will fail, because `getInstance()` returns a `DisposableInstance`, not the `BuilderImageWrapper` singleton.

Reply:
Imho, it still returns the same instance, doesn't it?  But yes now it needs type conversion like `as BuilderImageWrapper` in order to access `getBuilder()` and/or others `BuilderImageWrapper`'s properties/methods...

Reply:
Fixed.